src/Compilers/CSharp/Test/Semantic/Semantics/ExpressionBodiedMemberTests.cs

```diff
@@ -382,5 +382,354 @@ class Program
                 //     private void M() => (new object());
                 Diagnostic(ErrorCode.ERR_IllegalStatement, "(new object())").WithLocation(4, 25));
         }
+
+        [Fact, WorkItem(1702, "https://github.com/dotnet/roslyn/issues/1702")]
```

Comment:
I recommend simplifying things by changing the syntax tree API by adding `ExpressionBody` to `BaseMethodDeclarationSyntax`. It looks like an oversight that it isn't that way today.